optimize/components/userguide/edit-mode.md

```diff
@@ -34,7 +34,11 @@ In the **Add a Tile** modal, choose **Text** to access the text editor. This ena
 
 ![text editor](./img/dashboard-addAReportModal-textReport.png)
 
-### Placing tile on the dashboard
+## Copying, Editing, and Removing Tiles
+
+Each tile features a set of buttons in its top-right corner, providing you with options to edit and remove the tile. The 'Copy' button allows you to duplicate any tile, and the copied version can be conveniently placed anywhere on the dashboard. This duplicated tile is then customizable, enabling you to tailor it to your specific requirements.
```

Comment:
I would say the "copy icon" here. You could even add a microimage of this, for example `<img src={ModelerGlobalNavImg} style={{width: 200}} alt="Navigation to Web Modeler in the global nav" />`

Reply:
I like the idea of microimages in the text. I didn't know we can do this. I will give it a try ;)